README.md

```diff
@@ -1,12 +1,12 @@
 Marek's approach to building AMD GPU drivers for driver development
 ===================================================================
 
-These instructions have only been tested on Ubuntu 20.04.
+These instructions have been tested on Ubuntu 22.04.
 
 You are going to need the following packages:
 
 ```bash
-sudo apt install git make gcc flex bison libncurses-dev libssl-dev libelf-dev libelf-dev:i386 libzstd-dev libzstd-dev:i386 zstd python3-setuptools libpciaccess-dev libpciaccess-dev:i386 ninja-build libcairo2-dev libcairo2-dev:i386 gcc-multilib cmake-curses-gui g++ g++-multilib ccache libudev-dev libudev-dev:i386 libglvnd-dev libglvnd-dev:i386 libxml2-dev libxml2-dev:i386 graphviz doxygen xsltproc xmlto xorg-dev libxcb-glx0-dev libxcb-glx0-dev:i386 libx11-xcb-dev libx11-xcb-dev:i386 libxcb-dri2-0-dev libxcb-dri2-0-dev:i386 libxcb-dri3-dev libxcb-dri3-dev:i386 libxcb-present-dev libxcb-present-dev:i386 libxshmfence-dev libxshmfence-dev:i386 libxfixes-dev:i386 libxxf86vm-dev:i386 libxkbcommon-dev libvulkan-dev spirv-tools glslang-tools python3-numpy libcaca-dev python3-lxml autoconf libtool automake xutils-dev
+sudo apt install git make gcc flex bison libncurses-dev libelf-dev libzstd-dev libzstd-dev:i386 zstd python3-setuptools libpciaccess-dev ninja-build libcairo2-dev libva-dev gcc-multilib cmake-curses-gui g++ g++-multilib ccache libudev-dev libudev-dev:i386 libglvnd-dev libxml2-dev graphviz doxygen xsltproc xmlto xorg-dev wayland-protocols libwayland-egl-backend-dev libxcb-glx0-dev libxcb-glx0-dev:i386 libx11-xcb-dev libx11-xcb-dev:i386 libxcb-dri2-0-dev libxcb-dri2-0-dev:i386 libxcb-dri3-dev libxcb-dri3-dev:i386 libxcb-present-dev libxcb-present-dev:i386 libxshmfence-dev libxshmfence-dev:i386 libxfixes-dev libxfixes-dev:i386 libxxf86vm-dev libxxf86vm-dev:i386 libxrandr-dev libxkbcommon-dev libvulkan-dev spirv-tools glslang-tools python3-numpy libcaca-dev python3-lxml autoconf libtool automake xutils-dev vim meson build-essential
```

Comment:
We still need the libraries required for building on Ubuntu 20.04.

This file has instructions to install meson from source. Why is meson also included here?

build-essential is required for building debian packages according to the package description. We are not building debian packages though.

Why vim?